### PR TITLE
Global test properties

### LINKED ISSUE #11: Add submodule update to installer
Ensure proper version of OCF is included in repo by running:
git submodule update --init --recursive

### PR DESCRIPTION
**Pull request by katlapinka**
Added global test properties (dut and executor)
Simple dut representation class
Update for installer and example test
Closes #11

---
This change is [Reviewable](https://reviewable.io/reviews/open-cas/test-framework/9)

### CODE REVIEW

File: utils/dut.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        self.spider = dut_info['spiderip'] if 'spiderip' in dut_info else None
+        self.wps = dut_info['wpsid'] if 'wpsid' in dut_info else None
+        self.wps_port = dut_info['wpsport'] if 'wpsport' in dut_info else None
+
+
+    def __str__(self):
+        str =  f'ip: {self.ip}\n'
+        str += f'disks:\n'
+        for disk in self.disks:
+            str += f'\ttype: {disk["type"]}, serial: {disk["serial"]}, size: {disk["size"]}, blocksize: {disk["blocksize"]}\n'
```
Comment (KamilLepek) [inline utils/dut.py]: lets keep this under 100 characters, ill add some pep8 checker later on

File: utils/dut.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+class Dut:
+    def __init__(self, dut_info):
+        self.ip = dut_info['ip']
+        self.disks = dut_info['disks']
+        self.ipmi = dut_info['ipmiip'] if 'ipmiip' in dut_info else None
+        self.spider = dut_info['spiderip'] if 'spiderip' in dut_info else None
+        self.wps = dut_info['wpsid'] if 'wpsid' in dut_info else None
+        self.wps_port = dut_info['wpsport'] if 'wpsport' in dut_info else None
+
+
```
[the remark this answers is not in the document]
Comment (katlapinka) [reply utils/dut.py]: done

File: utils/dut.py (hunk 1 above)
Comment (katlapinka) [reply utils/dut.py]: done

File: installers/installer.py [hunk 3]
```diff
@@ -12,44 +13,45 @@
...
+    TestProperties.executor.execute(f"if [ -d {opencas_repo_name} ]; "
+                                    f"then rm -rf {opencas_repo_name}; fi")
+    output = TestProperties.executor.execute(
         "git clone --recursive https://github.com/Open-CAS/open-cas-linux.git")
     if output.exit_code != 0:
         raise Exception(f"Error while cloning repository: {output.stdout}\n{output.stderr}")
 
     LOGGER.info("Open CAS make and make install.")
-    output = executor.execute("cd open-cas-linux/ && ./configure && make")
+    output = TestProperties.executor.execute("cd open-cas-linux/ && ./configure && make")
```
Comment (Deixx) [inline installers/installer.py]: Could you address #11 as you're already modifying this?

File: utils/dut.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+
+class Dut:
+    def __init__(self, dut_info):
+        self.ip = dut_info['ip']
+        self.disks = dut_info['disks']
+        self.ipmi = dut_info['ipmiip'] if 'ipmiip' in dut_info else None
```
Comment (Deixx) [inline utils/dut.py]: could we add _ to separate ip/port in the names?

File: installers/installer.py (hunk 3 above)
Comment (katlapinka) [reply installers/installer.py]: done

File: utils/dut.py (hunk 4 above)
Comment (katlapinka) [reply utils/dut.py]: we discussed not to do it ;)

**Pull request merged**